orttraining/orttraining/python/training/ortmodule/experimental/hierarchical_ortmodule/_hierarchical_ortmodule.py

```diff
@@ -1,15 +1,22 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # debug_options.py
+import tempfile
 import torch
-from . import ORTModule
+from ... import ORTModule
 
 
-class HierarchalORTModule(torch.nn.Module):
+class HierarchicalORTModule(torch.nn.Module):
+    # This class recursively wraps sub-modules of "self._wrapped_module"
+    # as ORTModule whenever it can. The actual wrapping happens in its
+    # first "forward(...)" call because we need to call Pytorch-to-ONNX
+    # exporter with actual inputs. Afterwards, supported computation
+    # will be delegated to ORT and unsupported computation is still
+    # done by Pytorch.
     def __init__(self, module):
```

Comment:
Should the ctor take `debug_options` just like `ORTModule` does? Important for debugging, such as saving ONNX and changing log verbosity

Reply:
No problem.